cadence/src/client.rs

```diff
@@ -213,6 +213,50 @@ pub trait Histogrammed {
         -> MetricBuilder<'_, '_, Histogram>;
 }
 
+/// Trait for recording distribution values.
+///
+/// Similar to histrograms, but applies globally. A distrubtion can be used to
+/// instrument logical objects, like services, independently from the underlying
+/// hosts.
+///
+/// See the [Datadog docs](https://docs.datadoghq.com/developers/metrics/types/?tab=distribution#definition)
+/// for more information.
+///
+/// Note that tags and distributions are a
+/// [Datadog](https://docs.datadoghq.com/developers/dogstatsd/) extension to
+/// Statsd and may not be supported by your server.
+pub trait Distributed {
+    /// Record a single distribution value with the given key
+    fn distribution(&self, key: &str, value: u64) -> MetricResult<Distribution> {
+        self.distribution_with_tags(key, value).try_send()
+    }
+
+    /// Record a single distribution value with the given key and return a
+    /// `MetricBuilder` that can be used to add tags to the metric.
+    fn distribution_with_tags<'a>(&'a self, key: &'a str, value: u64) -> MetricBuilder<'_, '_, Distribution>;
+
+    /// Record a single distribution value with the given key.
```

Comment:
If you don't have a specific usecase for treating these as times, would you mind removing the `duration` methods? They were added to Histograms (in Cadence) after the fact by request. I'd rather not add extra methods unless people specifically need them.

Reply:
Will do!